triage-party/overlays/moc/smaug/secrets.enc.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+items:
```

Comment:
There's no need to keep the secret in a `List`, can this resource be of `Secret` kind please?

Reply:
Will do, thanks